src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -46,9 +51,13 @@
     public ResponseEntity<UserDto> updateUserInfo(@Valid @RequestBody UserDto userDto, Principal principal)
```

Comment:
You can change return type of this method to void, add @ResponceStatus.ACCEPTED annotation and delete return statement

Reply:
I agree with your suggestion about status, but don't about return type. After updating front-end dev can use new data without sending extra  GET requests to get them